client/invoice_table.js

```diff
@@ -1,17 +1,39 @@
+Template.InvoiceTable.onCreated(function () {
+  var instance = this;
+  limit = new ReactiveVar(20);
+
+  instance.autorun(function () {
+    instance.cursor = instance.subscribe('invoices', date.get(), days.get(), limit.get());
+  });
+});
+
 Template.InvoiceTable.helpers({
   invoices: function () {
-    return InvoiceTicketsCollection.find(filter.get(), {
+    return InvoiceTicketsCollection.find(getFilter(date.get(), days.get()), {
       sort: {
         createdAt: sortState.get('sortCreatedAt', true),
         total: sortState.get('sortTotal', true)
-      }
+      },
+      limit: limit.get()
     });
   },
   createdAtButtonText: function () {
     return sortState.get('sortCreatedAt');
   },
   totalButtonText: function () {
     return sortState.get('sortTotal');
+  },
+  dummy: function () {
+    return "Test";
+  },
+  hasMoreContent: function () {
+    return !(InvoiceTicketsCollection.find(getFilter(date.get(), days.get()), {
+      sort: {
+        createdAt: sortState.get('sortCreatedAt', true),
+        total: sortState.get('sortTotal', true)
+      },
+      limit: limit.get()
+    }).count() < limit.get());
```

Comment:
The interesting thing is that this part of the code together with the previously mentioned part will actually work as intended as long as
`numberOfInvoicesOnServer ≠ initiallyLoadedAmount + N × incrementalStep` :stuck_out_tongue_winking_eye: